tests/aws/services/dynamodb/test_dynamodb.py

```diff
@@ -746,6 +746,34 @@ def test_dynamodb_batch_execute_statement(
 
         aws_client.dynamodb.delete_table(TableName=table_name)
 
+    @markers.aws.validated
+    def test_dynamodb_execute_statement_empy_parameter(
+        self, dynamodb_create_table_with_parameters, snapshot, aws_client
+    ):
+        table_name = f"test_table_{short_uid()}"
+        dynamodb_create_table_with_parameters(
+            TableName=table_name,
+            KeySchema=[
+                {"AttributeName": "Artist", "KeyType": "HASH"},
+                {"AttributeName": "SongTitle", "KeyType": "RANGE"},
+            ],
+            AttributeDefinitions=[
+                {"AttributeName": "Artist", "AttributeType": "S"},
+                {"AttributeName": "SongTitle", "AttributeType": "S"},
+            ],
+            ProvisionedThroughput={"ReadCapacityUnits": 5, "WriteCapacityUnits": 5},
+        )
+
+        aws_client.dynamodb.put_item(
+            TableName=table_name,
+            Item={"Artist": {"S": "The Queen"}, "SongTitle": {"S": "Bohemian Rhapsody"}},
+        )
+
+        statement = f"SELECT * FROM {table_name}"
+        with pytest.raises(Exception) as e:
+            aws_client.dynamodb.execute_statement(Statement=statement, Parameters=[])
+        snapshot.match("invalid-param-error", e.value)
```

Comment:
nit: It would be good to validate that the exception is a `ClientError` and is raised from the provider and not from boto. For this, you can disable validation of the client to get the proper exception raised in AWS too: we have a few examples in our tests:
https://github.com/localstack/localstack/blob/41ffa76891286fffa5ca72a7b501fd7ecbbd36c4/tests/aws/services/s3/test_s3_list_operations.py#L804

`ddb_client = aws_client_factory(config=Config(parameter_validation=False)).dynamodb` would allow you to pass incorrect input and still get the AWS error.

Reply:
Awesome, thanks a lot for addressing the comment at light's speed! ❤️🚀

Reply:
Thank you for the nice suggestion 🚀 . I have never come across disabling parameter validation before.